knowledge-base/webforms-reports-do-not-display-changes-in-events.md

```diff
@@ -0,0 +1,41 @@
+---
+title: WebForms Reports Do Not Display Data or Applied Changes in Events
+page_title: WebForms Reports Do Not Display Data or Applied Changes in Events
+description: "Learn how to fix the error about reports that do not display data or applied changes in the events when using the ASP.NET WebForms ReportViewer."
+slug: webforms-reports-do-not-display-changes-in-events
+tags: telerik, reporting, troubleshoot, asp, net, webforms, reportviewer, reports, do, not, display, data, changes, in, events
+published: True
+type: troubleshooting
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Version</td>
+			<td>Q2 2012 and newer</td>
+		</tr>
+	        <tr>
+			<td>Report Viewers</td>
+			<td>WinForms, WPF, ASP.NET</td>
```

Comment:
Related only to the legacy AAP.NET WebForms Viewer

Include a note about this so that it is clear to the readers. Most of them don't use this viewer anymore and we recommend them to migrate to the Html5 Viewer